src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/RuntimeMetrics.cs

```diff
@@ -188,17 +187,16 @@ private static IEnumerable<Measurement<long>> GetGarbageCollectionCounts()
             }
         }
 
-        // TODO - Uncomment once an implementation for https://github.com/dotnet/runtime/issues/104844 is available.
-        //private static IEnumerable<Measurement<double>> GetCpuTime()
-        //{
-        //    if (OperatingSystem.IsBrowser() || OperatingSystem.IsTvOS() || OperatingSystem.IsIOS())
-        //        yield break;
+        private static IEnumerable<Measurement<double>> GetCpuTime()
+        {
+           if (OperatingSystem.IsBrowser() || OperatingSystem.IsTvOS() || OperatingSystem.IsIOS())
```

Comment:
Should we just avoid publishing the counter at all on these platforms?

Reply:
Do you mean all exposed runtime counters? I cannot judge that here as I don't know which counters can work under these platforms.

Reply:
I mean rather than creating a counter that will never publish any values, instead not creating the counter at all.

Reply:
got it. Although this counter is not going to cause noticeable overhead, I agree with not creating it better (at least not doing the check every time we publish values). I'll change that. 

CC @noahfalk @stevejgordon